src/libraries/System.Runtime/tests/System/ByteTests.GenericMath.cs

```diff
@@ -116,6 +115,16 @@ public static void TrailingZeroCountTest()
             Assert.Equal((byte)0x00, BinaryIntegerHelper<byte>.TrailingZeroCount((byte)0xFF));
         }
 
+        [Fact]
+        public static void GetShortestBitLengthTest()
+        {
+            Assert.Equal(0x00, BinaryIntegerHelper<byte>.GetShortestBitLength((byte)0x00));
```

Comment:
I was originally confused by your use of hex on the left-hand side, but I can see why it helps with manually calculating the expected result (counting groups of 16 bits).